HW02_B_Mimic_starter.Rmd

```diff
@@ -42,26 +43,47 @@ data("diamonds")
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}
 
-
+ggplot(diamonds, aes(cut, fill = clarity))+
+  geom_bar(position = "dodge")+
+  labs(title="My Diamond Collection",
+       subtitle="Boxplot representing the number of diamonds in my diamon collection by\ntype of cut quality and clarity of diamond")+
+  theme(plot.title = element_text(hjust=0.5, face="bold"))+
+  xlab("Dimond Cut")+
+  ylab("Number of Diamonds")+
+  annotate("text", x="Premium", y=4500, label="My Best Diamonds\nof course")+
+  annotate("rect", xmin="Premium", xmax="Ideal", ymin=0, ymax=5000, alpha=0.5)# I don't know how to move the shade and text more to the left/right, since x-aix is discrete. 

```

Comment:
I achieved the same effect by using xmin=4.5, xmax=5.5. I guess we can use the same approach to move the shade to left or right.